tests/components/element-card/element-card.spec.tsx

```diff
@@ -107,3 +122,15 @@ const MOCK_ELEMENT = {
 };
 
 const mockOnClick = jest.fn();
+
+const mockDeleteElement = jest.fn();
+
+jest.mock("src/context-providers/builder/hook.ts", () => {
+    const actual = jest.requireActual("src/context-providers/builder/hook.ts");
+    return {
+        useBuilder: () => ({
+            ...actual.useBuilder(),
+            deleteElement: mockDeleteElement,
+        }),
+    };
+});
```

Comment:
logically speaking, the mocks are hoisted to the top of the module it is mocking so we place them as close to the imports as possible. hence pls move this block of codes above the main test suite, just below the imports